### PR TITLE
Feature/issue 58/how it works

### PR DESCRIPTION
**Pull request by LyssenkoAlex**
Add how-it works block to Merch block

### CODE REVIEW

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx [hunk 1]
```diff
@@ -1 +1,43 @@
-export const HowItWorks = () => <h1>How it works</h1>;
+import { urlForImage } from '@/lib/sanity';
+import Image from 'next/image';
+import { ImagesListItem } from '@/types';
+
+type Props = {
+  howItWorksImages: ImagesListItem[];
+  howItWorksBlock: string;
+};
```
Comment (alexey-koran) [inline components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: It's an interface, not a type

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx [hunk 2]
```diff
@@ -1 +1,43 @@
...
+            className="h-full w-full object-center group-hover:opacity-75"
+          />
+        </div>
+        <h2 className="text-center font-bold">{item.caption}</h2>
+        <p className="text-sm font-medium text-night-700 mt-4">
+          {item.description}
+        </p>
+      </div>
+    </div>
+  ));
```
Comment (alexey-koran) [inline components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: Render list inside JSX

**Review by alexey-koran — CHANGES_REQUESTED** [review]

File: src/pages/merch/index.tsx [hunk 3]
```diff
@@ -23,11 +23,13 @@ export default function Index({
...
   );
-
   return (
     <>
       <GeneralInfo merch={allMerchGeneral[0]?.general_info} />
-      <HowItWorks />
+      <HowItWorks
+        howItWorksImages={allMerchGeneral[0]?.howItWorksImages}
+        howItWorksBlock={allMerchGeneral[0]?.howItWorksBlock}
+      />
```
Comment (alexey-koran) [inline src/pages/merch/index.tsx]: Destructure it before usage

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx [hunk 4]
```diff
@@ -1 +1,43 @@
-export const HowItWorks = () => <h1>How it works</h1>;
+import { urlForImage } from '@/lib/sanity';
+import Image from 'next/image';
+import { ImagesListItem } from '@/types';
+
+type Props = {
+  howItWorksImages: ImagesListItem[];
+  howItWorksBlock: string;
```
Comment (alexey-koran) [inline components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: Use shorter components props names:
```suggestion
  images: ImagesListItem[];
  title: string;
```

**Review by alexey-koran — CHANGES_REQUESTED** [review]

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx (hunk 1 above)
Comment (LyssenkoAlex) [reply components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: I think it is a type, all other pages use type for this case

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx (hunk 2 above)
Comment (LyssenkoAlex) [reply components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: Should I rename file?

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx (hunk 4 above)
Comment (LyssenkoAlex) [reply components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: Did changes on DE side, still do not good tool to rename sanity schema

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx (hunk 1 above)
Comment (alexey-koran) [reply components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: > I think it is a type, all other pages use type for this case

So all the other pages use this wrong

File: components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx (hunk 2 above)
Comment (alexey-koran) [reply components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx]: > Should I rename file?

Not sure if this is needed

**Review by alexey-koran — APPROVED** [review]
Approved

**Pull request merged**